internal/governor/governor.go

```diff
@@ -146,11 +153,14 @@ func schedule(r *bufio.Reader, w io.Writer) error {
 				duration = time.Duration(d) * time.Second
 			}
 		}
-		return newWaitError(duration)
+		if len(words) > 2 {
+			reason = words[2]
```

Comment:
I'm confused about why wait and fail handle the rest of the words differently. Why does wait only get one reason, but fail gets any number of reasons? Would it make sense for this to be more like the "fail" case and join all the remaining strings? Even if we don't return multiple strings now, it might be useful to do so in the future.

```suggestion
			reason = strings.Join(words[2:], " ")
```